### ISSUE TITLE
Rage flag causing unhandled CLI query error

### ISSUE THREAD

**Issue by yassere** [romefrontend/rome#789]
## Steps To Reproduce

Run `./scripts/dev-rome check --rage`

Rome exits with:
```
Unhandled CLI query error
Error: rage() can only be called before a query has been dispatched
```

## Expected Results

Rome should complete the command and create a tarball containing debugging information.

**Comment by sebmck** [comment]
The condition on line 315 in `Client.ts` should be inverted.

**Comment by sebmck** [comment]
Fixed by #791.
